app/controllers/api/venues_controller.rb

```diff
@@ -0,0 +1,41 @@
+# :reek:InstanceVariableAssumption
+# rubocop:disable Style/ClassAndModuleChildren
+class Api::VenuesController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
hmm.) I know that I'm not aware of project details, but this looks weird)

Reply:
removed